### ISSUE TITLE
Add Regex pattern testcases.

### ISSUE THREAD

**Issue by bjompen** [bjompen/PSSecretScanner#14]
I'm still missing patterns for the regex pattern tests for the following:

- _Cloudinary
- _Firebase_URL
- _Amazon_MWS_Auth_Token
- _Facebook_Access_Token
- _Facebook_OAuth
- _GitHub
- _Generic_API_Key
- _Generic_Secret
- _Google_API_Key
- _Google_Cloud_Platform_OAuth- :
- _Google_GCP_Service_account
- _Google_OAuth_Access_Token
- _Heroku_API_Key
- _MailChimp_API_Key
- _Mailgun_API_Key
- _Password_in_URL
- _PayPal_Braintree_Access_Token
- _Picatic_API_Key
- _Slack_Webhook
- _Stripe_API_Key
- _Stripe_Restricted_API_Key
- _Square_Access_Token
- _Square_OAuth_Secret
- _Twilio_API_Key
- _Twitter_Access_Token
- _Twitter_OAuth

If possible I wan't to be able to add matching patterns, and non matching patterns, but just the general look of these tokens are better than nothing.

Basically, all from the h33tlit list. If anyone has _obfuscated_ patterns to share, please do so, or add them to the pattern testcases.
